### ISSUE TITLE
[Bug] Unexpected keyword argument for highlight_rows and highlight_cols

### ISSUE THREAD

**Issue by markhlang** [ragardner/tksheet#24]
When calling `highlight_rows` or `highlight_cols` they are  calling their respective functions in tksheet_main_table.  However those don't expect the argument `highlight_index` but instead want `highlight_header`.

```python
Traceback (most recent call last):
  File "$WORKFOLDER/code/fiprotima/src/test_test.py", line 55, in <module>
    sheet.highlight_rows()
  File "~/.local/share/virtualenvs/fiprotima-B_NhjLga/lib/python3.8/site-packages/tksheet/_tksheet.py", line 1047, in highlight_rows
    self.MT.highlight_rows(rows = rows,
TypeError: highlight_rows() got an unexpected keyword argument 'highlight_index'
```

**Comment by ragardner** [comment]
@markhlang Sorry about that and thanks very much for pointing it out, it should be fixed now in version `4.8.3`

### Version 4.8.3
 - Make `enable_bindings()` and `disable_bindings()` without arguments do all bindings
 - Fix top left resizers not showing up if enabling/disabling all bindings
 - Improve look and feel of drag and drop
 - Fix bug in certain circumstances with `insert_row_positions()`
 - Fix `highlight_rows()` bug

**Comment by markhlang** [comment]
Works as expected. Thank you.
